spec/support/taggable.rb

```diff
@@ -0,0 +1,24 @@
+RSpec.shared_examples "taggable" do
+  it 'returns empty tags with no tag taggables' do
+    taggable = create(taggable_model, tags_count: 0)
+    expect(taggable.tags).to be_empty
+  end
+
+  it 'returns all tags with many tag taggables' do
+    taggable = create(taggable_model)
+    expect(taggable.tags).to_not be_empty
+    expect(taggable.tags).to eq(taggable.tag_taggables.map { |tag_taggable| tag_taggable.tag })
+  end
+
+  it 'on destroy destroys tag taggables, not taggables' do
```

Comment:
I wonder whether it would be advisable to put an underscore in `tag taggables` in these `it` descriptions so that they are easier to digest as a single entity?